apps/src/lab2/views/dialogs/DialogControlContext.ts

```diff
@@ -0,0 +1,14 @@
+import React, {useContext} from 'react';
+
+import {TypedDialogProps, closeDialogType} from './types';
+
+interface DialogControl {
+  closeDialog: closeDialogType;
+  showDialog: (args: TypedDialogProps) => void;
+}
+
+export const DialogControlContext = React.createContext<DialogControl>({
+  closeDialog: () => {},
+  showDialog: () => {},
+});
+export const useDialogControl = () => useContext(DialogControlContext);
```

Comment:
This file is really just moving the `DialogContext` out of `DialogManager` into a separate file, and renaming to `DialogControlContext` for clarity. It also gets a wrapper.

Also, `closeDialog` is now exposed as a function in the context as well.